modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-apply-policies_{context}"]
+= Applying update policies to managed clusters
+
+The `ClusterGroupUpgrade` CR notifies the Topology-Aware Lifecycle Operator to update the current cluster version to the version specified in the update policy. This action changes the status of the {rh-rhacm} policy from `inform` to `enforce`.
+
+.Prerequisites
+
+* Install the Topology-Aware Lifecycle Operator.
+* Provision one or more managed SNO clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CR, and then save the YAML in the `cgu-cr.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4915
+  namespace: ztp-group-du-sno
+spec:
+  deleteObjectsOnCompletion: true
+  Clusters: <1>
+  - cnfdb1
+  - cnfdb2
+  enable: true
+  managedPolicies:
+  - du-upgrade-platform-upgrade <2>
+  remediationStrategy:
+    maxConcurrency: 2 <3>
+    timeout: 240 <4>
+----
+<1> The list of clusters to update.
+<2> The name of the policy to enforce.
+<3> The `maxConcurrency` field signifies the number of clusters updated at the same time.
+<4> The update timeout in minutes.
+
+. Create the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f cgu-cr.yaml
+----
+
+.Verification
+
+. Check if the `ClusterGroupUpgrade` CR is created by running the following command:
++
+[source,terminal]
+----
+$ oc get cgu -A
+----
++
+.Example output
++
+[source,terminal]
+----
+cnfdb1                ztp-group-du-sno.du-upgrade-platform-upgrade                   inform               Compliant          3d19h
+cnfdb2                ztp-group-du-sno.du-upgrade-4915-du-upgrade-platform-upgrade   enforce              Compliant          23s <1>
+cnfdb2                ztp-group-du-sno.du-upgrade-platform-upgrade                   inform               NonCompliant       3d19h
+ztp-group-du-sno      du-upgrade-4915-du-upgrade-platform-upgrade                    enforce             Compliant          24s <1>
+ztp-group-du-sno      du-upgrade-platform-upgrade                                    inform               NonCompliant       3d19h
+----
+<1> The status of the targeted clusters is changed to enforce.
```

Comment:
I don't understand what we're trying to show in lines 71 to 88.

Reply:
The integration tests from TALO have a bunch of complete examples: https://github.com/openshift-kni/cluster-group-upgrades-operator/tree/main/tests/kuttl/tests